src/components/organisms/NewResourceWizard/NewResourceWizard.tsx

```diff
@@ -91,6 +129,41 @@ const NewResourceWizard = () => {
     [registeredKindHandlers, resourceMap]
   );
 
+  const generateExportFileName = async () => {
+    if (fileMap[ROOT_FILE_ENTRY] && selectedFolder.startsWith(fileMap[ROOT_FILE_ENTRY].filePath)) {
+      const currentFolder = selectedFolder.split(`${fileMap[ROOT_FILE_ENTRY].filePath}`).pop();
+
+      if (currentFolder) {
+        setSelectedFolder(currentFolder.slice(1));
+      } else {
+        setSelectedFolder(ROOT_FILE_ENTRY);
+      }
+      return;
+    }
+
+    let selectedFolderResources;
+    if (selectedFolder === ROOT_FILE_ENTRY) {
+      selectedFolderResources = Object.values(resourceMap).filter(
+        resource => resource.filePath.split('/').length === 2
```

Comment:
We should use `path.sep` instead of `/` because it is different on each OS system.